lua/leetcode-ui/question.lua

```diff
@@ -19,13 +19,36 @@ local log = require("leetcode.logger")
 local Question = Object("LeetQuestion")
 
 function Question:get_snippet()
+    local inject = config.user.inject
+    -- If inject.start is a string, assign to inject_start
+    -- else if inject.start is a string[], join into inject_start with "\n" as separator
+    local inject_before = ""
+    local inject_after = ""
+
+    if type(inject.before) == "string" then
+        inject_before = inject.before
+    else
+        inject_before = table.concat(inject.before, "\n")
+    end
+    if type(inject.after) == "string" then
+        inject_after = inject.after
+    else
+        inject_after = table.concat(inject.after, "\n")
+    end
```

Comment:
modify accordingly to the other comment so it will get the inject for current question language.

Reply:
and also you will have to handle when after or before and nil